biogeophys/FatesHydroWTFMod.F90

```diff
@@ -751,8 +754,12 @@ function th_from_psi_cch(this,psi) result(th)
     if(psi>this%psi_max) then
         ! Linear range for extreme values
         th = this%th_max + (psi-this%psi_max)/this%dpsidth_max
-    else
-        th = this%th_sat*(psi/this%psi_sat)**(-1.0_r8/this%beta)
+     else
+        if(psi<this%psi_min) then
+           th = this%th_min
```

Comment:
Wouldn't this artificially add water to soils?

Reply:
yes, but the intention is to go from a value of zero to an inconsequentially small non-zero value. @ckoven and I were talking, this could potentially be problematic if we artificially add small amounts of water and that enables or drives non-zero fluxes that would had otherwise been zero. One possibility would be to force fractional conductivity to zero where it hits theta min.  That seems messy though. Honestly, I think if we just use a suitably small minimum value, one that is an incredibly small water content, yet can still accomodate the math needed to switch between psi and theta, this should be sufficient. Perhaps we can try minimum capping at -15 MPa or smaller, perhaps -9 is too high.

Reply:
I don't believe this is a mass conservation liability. We use these functions to drive fluxes, they never actually over-write the original masses of each pool.  It is the fluxes that update the pools.  What will happen is that water contents below the psi_min or theta_min values will just end up having zero flux out of that pool, and thus remain unchanged.

For instance here is where we update the actual soil water content, it is based off of fluxes:
https://github.com/NGEET/fates/blob/main/biogeophys/FatesPlantHydraulicsMod.F90#L2729-L2730